django_netjsonconfig/static/django-netjsonconfig/css/admin.css

```diff
@@ -96,8 +96,8 @@ a.jsoneditor-poweredBy:focus{ text-decoration: none }
 }
 #container .help{ margin-top: 2px; font-size: 13px }
 #container .help a,
-#netjsonconfig-hint a{ text-decoration: underline }
-#netjsonconfig-hint{ width: auto }
+#netjsonconfig-hint a, #netjsonconfig-hint-1 a{ text-decoration: underline }
```

Comment:
`netjsonconfig-hint-1` is not a very good name. Maybe you could use something like `netjsonconfig-hint-advancedmode` or `netjsonconfig-hint-ace`.